fastlane/Fastfile

```diff
@@ -380,16 +384,16 @@ end
     File.rename(File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote.ipa'), File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote Internal.ipa'))
 
     appcenter_upload(
-      api_token: ENV["APPCENTER_API_TOKEN"],
-      owner_name: "automattic",
-      owner_type: "organization",
-      app_name: "Simplenote",
+      api_token: ENV.fetch('APPCENTER_API_TOKEN', nil),
```

Comment:
Since it's a required param, we might prefer to use `get_required_env('APPCENTER_API_TOKEN')` here? 🤔

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/0ed00e9856a85c4d55b3045f7a510b8619015175